libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -313,6 +313,9 @@ void AP_Airspeed::init()
             delete sensor[i];
             sensor[i] = nullptr;
         }
+        if (sensor[i] != nullptr) {
```

Comment:
these changes look like copied from my PR, should have been done with cherry-pick to preserve author (and blame!)

Reply:
cherry picked 2 of your commits for baro and airspeed that are used unchanged in this PR. Thanks for the advice, will keep mind of it in future.